tools/build-tools/src/common/fileHashCache.ts

```diff
@@ -25,4 +25,4 @@ export class FileHashCache {
     public clear() {
         this.fileHashCache.clear();
     }
-};
\ No newline at end of file
+}
```

Comment:
Nit: Is EOF newline not turned on here?

Reply:
No. Our shared config is not trivial to enable on a package that has never been linted, and it's not easy to adopt only the formatting rules. I've talked with @Josmithr about possible ways we could address this, but in the meantime, I want to be pragmatic and enable some baseline protection.